vendor/browan/ambientlightsensor-codec.yaml

```diff
@@ -0,0 +1,33 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: ambientlightsensor.js
+  # Examples (optional)
+  format:
+    - description: ambient light sensor
+      input:
+        fPort: 104
+        bytes: [0, 5]
+      output:
```

Comment:
this is an example output, so the output needs to be human readable JSON formed from the example input. Your example input is
```
input:
         fPort: 104
         bytes: [0, 5]
```

so your output should be something like

```
output:
  data:
    darker: true // input.bytes[0] & 0x01
    lighter: false // input.bytes[0] & 0x02
    ...
    battery: 0 // input.bytes[1] & 0x07
    ...
    // and so on
```